src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -93,6 +95,40 @@ def transform_deployments_list(result):
     sort_list = sorted(result, key=lambda deployment: deployment['properties']['timestamp'])
     return [transform_deployment(r) for r in sort_list]
 
+def transform_stacks(result):
+    r = result
+    return OrderedDict([('Name', r['name']),
+                        ('State', r['provisioningState']),
+                        ('Last Modified', r['systemData']['lastModifiedAt']),
+                        ('Deployment Id', r['deploymentId'])])
```

Comment:
This variable replacement could be removed, instead using `result` variable directly.